acceptance/testdata/launcher/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN go build -o helper ./src/exec.d
 
 FROM ubuntu:bionic
 
-COPY container /
+COPY posix/container /
```

Comment:
Also not necessary for this PR but I probably should have named this directory `linux` when I originally introduced the launcher acceptance tests. It would be significantly less confusing.